eoscostarica/eosio-monitor
# add basic configuration

**What does this PR do?**
Create the base project with react, redux and material

**How should this be tested?**

`yarn`
`yar start`
then navigate to http://localhost:3000 and check the block producers
